### PR TITLE
New Fields: Applicant Rating

### PR DESCRIPTION
**Pull request by simonbbaumgartner**
Our service transfers data on the rating/solvency (CRIF Score) to the FI. We would like to have relevant fields in the object "Applicant":

rating
description Solvency rating of the applicant
type: integer

ratingDate
description: Date of the solvency rating
type: see Date

### CODE REVIEW

**Review by olegkovalov — APPROVED** [review]

File: mortgageAPI.yaml [hunk 1]
```diff
@@ -2608,6 +2608,19 @@ components:
               type: string
               description: Email of the applicant
               example: max.muster@domain.ch
+            rating:
+              type: integer
+              description: 'Solvency rating of the applicant e.g. CRIF'
+              example: '420'
```
Comment (stjosh) [inline mortgageAPI.yaml:2614]: This is a number, not a string, so the single quotes need to be removed

**Review by stjosh — CHANGES_REQUESTED** [review]

File: mortgageAPI.yaml (hunk 1 above)
Comment (simonbbaumgartner) [reply mortgageAPI.yaml]: done, see below

**Review by rolfwagner — CHANGES_REQUESTED** [review]
Could you please cluster these three new fields in a object:
solvencyRating
--> ratingValue
--> ratingSourceType
--> ratingDate

**Comment by simonbbaumgartner** [comment]
requested changes are done

**Review by rolfwagner — APPROVED** [review]
Perfect ;-) Thank you

**Pull request merged**